libr/magic/softmagic.c

```diff
@@ -662,7 +662,7 @@ static int mconvert(RMagic *ms, struct r_magic *m) {
 	case FILE_BEDATE:
 	case FILE_BELDATE:
 		p->l = (st32)
-		    ((p->hl[0]<<24)|(p->hl[1]<<16)|(p->hl[2]<<8)|(p->hl[3]));
+		    (((ut32)p->hl[0]<<24)|((ut32)p->hl[1]<<16)|((ut32)p->hl[2]<<8)|((ut32)p->hl[3]));
```

Comment:
It's a different order. Not here. The rest files - yes, I'm converting to r_read_* functions.